roles/shiftstack/tasks/pre_test_shiftstack.yml

```
@@ -13,6 +13,13 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
+- name: Remove the shiftstackclient pod if exists
+  kubernetes.core.k8s:
+    state: absent
+    api_version: v1
+    kind: Pod
+    namespace: "{{ cifmw_shiftstack_client_pod_namespace }}"
+    name: "{{ cifmw_shiftstack_client_pod_name + '_pod.yml' }}"
```

```suggestion
    name: "{{ cifmw_shiftstack_client_pod_name }}"
```
I think we should use the pod name here.

@eurijon wdyt about adding a var for the `cifmw_shiftstack_client_pod_name_pod.yml` file?

---

We can add a param for that indeed (same would apply for the PVC yaml file). The appended `_pod.yml` and `_pvc.yaml` strings are just for the yml files names, so we can easily identify each file when looking at the artifacts. Having them in params can make the code cleaner.